innopoints/views/notification.py

```diff
@@ -0,0 +1,35 @@
+"""Views related to notifications.
+
+- GET   /notifications
+- PATCH /notifications/{notification_id}/read
+"""
+
+from flask import abort
+from flask_login import login_required, current_user
+
+from innopoints.blueprints import api
+from innopoints.extensions import db
+from innopoints.models import Notification
+from innopoints.schemas import NotificationSchema
+
+
+@api.route('/notifications')
+@login_required
+def get_notifications():
+    """Gets all notifications of the current user."""
+    query = Notification.query.filter_by(recipient_email=current_user.email)
+    notifications = query.all()
+    schema = NotificationSchema(many=True)
+    return schema.jsonify(notifications)
+
+@api.route('/notifications/<int:notification_id>/read', methods=['PATCH'])
+@login_required
+def read_notification(notification_id):
+    """Marks a notification as read."""
+    notification = Notification.query.get_or_404(notification_id)
+    if notification.recipient_email != current_user.email:
+        abort(401, {'message': 'Not your notification'})
```

Comment:
I didn't pass messages to 401's, and I don't think we should. It's all clear from the error code

Reply:
well, it does no harm. think of it as an easter egg)
or do you care about being consistent that I should remove it?

Reply:
Well, it just kinda bugs me and I'll likely remove it next time I encounter it for consistency sake, but for now you may mark it as resolved and not bother much